src/server/req_delete.c

```diff
@@ -201,103 +201,169 @@ acct_del_write(char *jid, job *pjob, struct batch_request *preq, int nomail)
  *		done by req_deletejob()
  *
  * @param[in]	preq	- Batch request structure.
+ * @param[in]	pjob	- pointer to job
+ * @param[in]	parent	- pointer to parent job for sub job
```

Comment:
Remove the space between "sub" and "job".

Reply:
fixed in rc 5